packages/s3-store/index.ts

```diff
@@ -741,4 +799,29 @@ export class S3Store extends DataStore {
 
     return deleted
   }
+
+  private async uniqueTmpFileName(template: string): Promise<string> {
```

Comment:
Is there also a public package that we could reuse here?

Reply:
i think this is such a basic functionality that a package might be not necessary, wdyt?